cdap-ui/app/cdap/components/PipelineConfigurations/ConfigurationsContent/ResourcesTabContent/index.js

```diff
@@ -43,38 +44,38 @@ const StepContentHeading = ({ isBatch, engine }) => {
 };
 
 StepContentHeading.propTypes = {
-  isBatch: PropTypes.bool,
+  pipelineType: PropTypes.string,
   engine: PropTypes.string,
 };
 
 const ConnectedStepContentHeading = connect(mapStateToStepContentHeadingProps)(StepContentHeading);
 
-function ResourcesTabContent({ isBatch }) {
+function ResourcesTabContent({ pipelineType }) {
   return (
     <div
       id="resources-tab-content"
       className={classnames('configuration-step-content', {
-        'batch-content': isBatch,
-        'realtime-content': !isBatch,
+        'batch-content': GLOBALS.etlBatchPipelines.indexOf(pipelineType) !== -1,
+        'realtime-content': GLOBALS.etlBatchPipelines.indexOf(pipelineType) === -1,
       })}
     >
-      <ConnectedStepContentHeading isBatch={isBatch} />
+      <ConnectedStepContentHeading pipelineType={pipelineType} />
       <div className="resource-container">
-        {!isBatch ? <ClientResources /> : null}
+        {GLOBALS.etlBatchPipelines.indexOf(pipelineType) === -1 ? <ClientResources /> : null}
```

Comment:
since this is being used several times, better to extract to an `isBatch` variable before the `return`

Reply:
Fixed

Reply:
+1